docs/helm.md

```diff
@@ -50,7 +50,7 @@ variable name `resource`. The resource has the following fields available on it:
 Given this custom resource
 
 ```yaml
-apiVersion: stable.nicolerenee.io/v1
+apiVersion: stable.foobar.io/v1
```

Comment:
if we are going to change this it shouldn't be just some random url.

Reply:
I usually don't put authors name in the code/doc unless needed, reason why I updated it when I added a new section with similar ymal block. (reverted back).